src/Ctype/Ctype.php

```diff
@@ -219,14 +229,21 @@ private static function convert_int_to_char_for_ctype($int, $function)
             return (string) $int;
         }
 
-        if (\PHP_VERSION_ID >= 80100) {
-            @trigger_error($function.'(): Argument of type int will be interpreted as string in the future', \E_USER_DEPRECATED);
-        }
-
         if ($int < 0) {
             $int += 256;
         }
 
         return \chr($int);
     }
+
+    /**
+     * @param mixed $input
+     * @param string $function
+     */
+    public static function checkType($input, $function)
```

Comment:
Why is this method public?

```suggestion
     */
    private static function checkType($input, string $function): void
```